python/ray/tune/tests/test_trial_runner_pg.py

```diff
@@ -120,25 +118,14 @@ def on_step_end(self, iteration, trials, **info):
                     scheduled,
                     min(scheduled, len(trials)),
                     msg=f"Num trials iter {iteration}")
-
-                # The following two tests were relaxed for reuse_actors=True
-                # so that up to `max_num_parallel` more placement groups can
-                # exist than we would expect. This is because caching
-                # relies on reconciliation for cleanup to avoid overscheduling
-                # of new placement groups.
-                num_parallel_reuse = int(reuse_actors) * max_num_parallel
```

Comment:
Why do we change this?

Reply:
it's included accidentially. when I'm testing the revert.